src/ui/actions/timeline.ts

```diff
@@ -264,6 +269,44 @@ export function seek(
   };
 }
 
+export function jumpToReduxCode(
+  point: ExecutionPoint,
+  time: number,
+  sourcesState: SourcesState
+): UIThunkAction {
+  return async (dispatch, _, { replayClient }) => {
+    dispatch(framePositionsCleared());
```

Comment:
:question: Why do we need `framePositionsCleared` dispatched here?

Reply:
Hm I think we don't need it here since `seek` calls it internally. Looks like I forgot to remove it